orchestrator/graphql/utils/get_subscription_product_blocks.py

```diff
@@ -67,21 +67,29 @@ async def get_subscription_product_blocks(
 
     def to_product_block(product_block: dict[str, Any]) -> ProductBlockInstance:
         def is_resource_type(candidate: Any) -> bool:
-            return not isinstance(candidate, (list, dict))
+            return not isinstance(candidate, dict)
 
         def requested_resource_type(key: str) -> bool:
             return not product_block_instance_values or key in product_block_instance_values
 
         def included(key: str, value: Any) -> bool:
             return is_resource_type(value) and requested_resource_type(key) and key not in pb_instance_property_keys
 
+        def value_parser(value: Any) -> str:
+            if isinstance(value, (str, int, float, type(None))):
+                return value
+            elif isinstance(value, list):
+                return ", ".join([str(v) for v in value if is_resource_type(v)])
```

Comment:
Perhaps we can just give the list back as an array, so the frontend (or other consuming application) can decide how to format it?

```suggestion
            elif isinstance(value, list):
                return [value_parser(v) for v in value if is_resource_type(v)]
```

Unless there's a reason that this should be a string (@tjeerddie  ?)

Reply:
nope, you're correct, this gives a better representation of the data and the frontend can visualize it however it wants

Reply:
@Mark90 and @tjeerddie fixed and pushed changes